### PR TITLE
sans-serif font removed

### PR DESCRIPTION
**Pull request by rajasekharreddy-sagamreddy**

### CODE REVIEW

File: static/css/objects.css [hunk 1]
```diff
@@ -140,6 +140,20 @@
...
+.parent-element {
+  -webkit-transform-style: preserve-3d;
+  -moz-transform-style: preserve-3d;
+  transform-style: preserve-3d;
+}
+
+.element {
+  position: center;
+  top: 100%;
+  transform: translateY();
```
Comment (jace) [inline static/css/objects.css:153]: What's the 3D transform for?

File: static/css/typography.css [hunk 2]
```diff
@@ -12,13 +11,15 @@ i,
 em,
 .prose-type     { font-family: "Adelle", serif; }
 */
+@import url(http://fonts.googleapis.com/css?family=Open+Sans);
 @import url(http://fonts.googleapis.com/css?family=Varela+Round);
-@import url(http://fonts.googleapis.com/css?family=Varela);
+@import url(http://fonts.googleapis.com/css?family=Varela);  font-family: , sans-serif;
```
Comment (jace) [inline static/css/typography.css]: This seems out of place. Also, do you need Varela and Open Sans? (Source Sans Pro is required for the networkbar, not Open Sans.)

File: static/css/typography.css [hunk 3]
```diff
@@ -12,13 +11,15 @@ i,
...
 body,
-.title-type     { font-family: "Varela Round",  "Gill Sans", "Gill Sans MT", sans-serif; }
+.title-type     { font-family: "Varela Round" , 'Varela Round' , 'sans-serif';}
```
Comment (jace) [inline static/css/typography.css]: sans-serif doesn't need quoting since it's a single word.

File: static/css/typography.css [hunk 4]
```diff
@@ -83,7 +84,7 @@ abbr {
 .unordered      { list-style-type: none; }
 
 .amp  {
-    font-family:'Varela Round' 'Hoefler Text', 'Book Antiqua','URW Palladio L';
+    font-family:'Varela Round' 'Hoefler Text', 'Book Antiqua',;
```
Comment (jace) [inline static/css/typography.css]: Remove Hoefler Text and Book Antiqua and add sans-serif. One of sans-serif or serif or monospace must always be present as the last item in a font-family definition.

File: static/css/typography.css [hunk 5]
```diff
@@ -127,7 +128,7 @@ abbr {
 }
 
 .navbar .nav>li>a, .navbar-brand{
-  font-family: 'Varela Round', sans-serif;
+  font-family: 'Varela Round';
```
Comment (jace) [inline static/css/typography.css]: Add sans-serif.

File: static/font/OFL.txt [hunk 6]
```diff
@@ -0,0 +1,93 @@
+Copyright (c) 2011, Joe Prince, Admix Designs (http://www.admixdesigns.com/),
```
Comment (jace) [inline static/font/OFL.txt]: Why is this file in our repo? We're not distributing fonts in this folder.

File: templates/layouts/page.html [hunk 7]
```diff
@@ -31,6 +31,7 @@
 <!--[if lte IE 8]>
 <link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/leaflet/0.6.3/leaflet.ie.css" />
 <![endif]-->
+<link href="//fonts.googleapis.com/css?family=Varela+Round:400" rel="stylesheet" type="text/css">
```
Comment (jace) [inline templates/layouts/page.html]: You have Varela Round twice now. The next line is also for Varela Round.

File: templates/pages/home.html [hunk 8]
```diff
@@ -21,9 +21,11 @@
 {% endblock %}
 
 {% block banner %}
-<header> 
-<div class="img_responsive">
+<header id="header-image">
+
+<div class="header-image">
```
Comment (jace) [inline templates/pages/home.html:26]: A class and id having the same name is recipe for confusion later on. Change one name.

File: static/css/objects.css (hunk 1 above)
Comment (rajasekharreddy-sagamreddy) [inline static/css/objects.css:153]: for home page banner image

File: static/css/typography.css [hunk 9]
```diff
@@ -12,13 +11,15 @@ i,
 em,
 .prose-type     { font-family: "Adelle", serif; }
 */
+
 @import url(http://fonts.googleapis.com/css?family=Varela+Round);
 @import url(http://fonts.googleapis.com/css?family=Varela);
```
Comment (jace) [inline static/css/typography.css]: Varela is unused and you're still missing Source Sans Pro here.

File: static/css/typography.css [hunk 10]
```diff
@@ -12,13 +11,15 @@ i,
 em,
 .prose-type     { font-family: "Adelle", serif; }
```
Comment (jace) [inline static/css/typography.css]: What's Adelle? Where is this used?

File: static/css/typography.css [hunk 11]
```diff
@@ -127,7 +128,7 @@ abbr {
 }
 
 .navbar .nav>li>a, .navbar-brand{
-  font-family: 'Varela Round', sans-serif;
+  font-family: 'Varela Round' sans-serif;
```
Comment (jace) [inline static/css/typography.css]: Comma missing.

File: static/css/typography.css [hunk 12]
```diff
@@ -2,24 +2,17 @@
...
-.title-type     { font-family: "Adelle", sans-serif; }
+/*body*/
 
-i,
-em,
-.prose-type     { font-family: "Adelle", serif; }
-*/
 @import url(http://fonts.googleapis.com/css?family=Varela+Round);
-@import url(http://fonts.googleapis.com/css?family=Varela);
+@import url(http://fonts.googleapis.com/css?family=Source+Sans+Pro);
```
Comment (jace) [inline static/css/typography.css]: URL should be `//fonts.googleapis.com/css?family=Source+Sans+Pro:400italic,600italic,400,600`

Remove `http:` in the other URL as well.

**Pull request merged**